warehouse/forms.py

```diff
@@ -68,7 +68,7 @@ def __call__(self, form, field):
             msg = (
                 results["feedback"]["warning"]
                 if results["feedback"]["warning"]
-                else "Password is too easily guessed."
+                else _("Password is too easily guessed.")
```

Comment:
I actually think we shouldn't translate this string, because it will be mixed with other non-localized strings from zxcvbn, which would be confusing.
```suggestion
                # Note: we can't localize this string because it will be mixed
                # with other non-localizable strings from zxcvbn
                else "Password is too easily guessed."
```